resources/views/partials/sidebar.blade.php

```diff
@@ -1,42 +1,51 @@
-<div class="bg-magenta-800 text-white w-80 flex flex-col flex-shrink-0">
-    <div class="h-20 flex items-center px-4 mb-8">
-        <img src="{{ asset('images/university-logo.png') }}" alt="DU Logo" class="h-12 mr-3">
-        <a href="/" class="inline-block logo leading-tight max-w-sm">
-            <h1 class="text-lg font-bold">Department of <br> Computer Science</h1>
-        </a>
-    </div>
-    <aside class="flex-1">
-        <ul class="font-bold text-white-90">
-            <li>
-                <a href="/" class="flex items-center py-2 px-4 hover:bg-magenta-700">
-                    <feather-icon name="grid" class="h-4 mr-2" stroke-width="2">Dashboard</feather-icon>
-                    Dashboard
+<sidebar-nav>
+    <template v-slot:default="data">
+        <div v-if="data.isvisible" class="bg-magenta-800 text-white sm:w-80 flex flex-col flex-shrink-0 h-full absolute sm:static z-50">
```

Comment:
when you have used kebab-cased properties to passs data into slots, here you can access using camelCase, i.e. `isVisible`